From a68687fa83b3c1528a911f0b43e8c22c56901ebe Mon Sep 17 00:00:00 2001
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Wed, 1 May 2013 17:40:06 +0100
Subject: [PATCH] Sync spec with Fedora

---
 virt-viewer.spec.in | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/virt-viewer.spec.in b/virt-viewer.spec.in
index e4caec1..cb0c9af 100644
--- a/virt-viewer.spec.in
+++ b/virt-viewer.spec.in
@@ -3,7 +3,7 @@
 # Default to skipping autoreconf.  Distros can change just this one line
 # (or provide a command-line override) if they backport any patches that
 # touch configure.ac or Makefile.am.
-%{!?enable_autotools:%define enable_autotools 1}
+%{!?enable_autotools:%define enable_autotools 0}
 
 %define with_gtk3 0
 %if 0%{?fedora} >= 15
@@ -16,13 +16,16 @@
 %endif
 
 %define with_govirt 0
+%if 0%{?fedora} >= 19
+%define with_govirt 1
+%endif
 
 %if 0%{?rhel} >= 6
 %define with_spice 1
 %endif
 
-# spice-gtk is x86 x86_64 only currently:
-%ifnarch %{ix86} x86_64
+# spice-gtk is x86 x86_64 arm only currently:
+%ifnarch %{ix86} x86_64 %{arm}
 %define with_spice 0
 %endif
 
@@ -71,6 +74,9 @@ BuildRequires: spice-protocol >= 0.10.1
 %endif
 BuildRequires: /usr/bin/pod2man
 BuildRequires: intltool
+%if %{with_govirt}
+BuildRequires: libgovirt-devel >= 0.0.3
+%endif
 
 %description
 Virtual Machine Viewer provides a graphical console client for connecting
@@ -85,9 +91,6 @@ the display, and libvirt for looking up VNC/SPICE server details.
 %if 0%{?enable_autotools}
 autoreconf -if
 %endif
-%if %{with_govirt}
-BuildRequires: libgovirt-devel >= 0.0.3
-%endif
 
 %if %{with_spice}
 %define spice_arg --with-spice-gtk
