From 2c1e7c18bff998f779c4a1159dda12e4d775f673 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Fri, 17 May 2013 16:51:14 +0200
Subject: [PATCH] build-sys: add debug-helper rule

---
 data/Makefile.am | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/data/Makefile.am b/data/Makefile.am
index 13835cc..a43374e 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -24,6 +24,12 @@ virt-viewer-$(VERSION).exe: virt-viewer.nsis deps.txt
 	makensis -NOCD -DDESTDIR=$$DESTDIR $< >/dev/null && \
 	rm -rf $$DESTDIR
 
+virt-viewer-debug-$(VERSION).exe: virt-viewer-debug.nsis
+	$(AM_V_GEN)DESTDIR=`mktemp -d` && \
+	make -C $(top_builddir) install DESTDIR=$$DESTDIR >/dev/null && \
+	makensis -NOCD -DDESTDIR=$$DESTDIR $< >/dev/null && \
+	rm -rf $$DESTDIR
+
 virt-viewer-$(WIXL_ARCH)-$(VERSION).msi: virt-viewer.wxs deps.txt
 	$(AM_V_GEN)DESTDIR=`mktemp -d` &&				\
 	make -C $(top_builddir) install DESTDIR=$$DESTDIR >/dev/null &&	\
